### PR TITLE
Integrate websdk

### PR DESCRIPTION
**Pull request by rynowak**

### CODE REVIEW

File: korebuild-lock.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-version:2.1.0-preview1-15679
-commithash:5347461137cb45a77ddcc0b55b2478092de43338
```
Comment (rynowak) [inline korebuild-lock.txt]: @pranavkm @natemcmaster is this right? What's the right way to update Korebuild/sdk?

File: src/Microsoft.VisualStudio.Mac.LanguageServices.Razor/Microsoft.VisualStudio.Mac.LanguageServices.Razor.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net461</TargetFrameworks>
+    <TargetFramework>net461</TargetFramework>
```
Comment (rynowak) [inline src/Microsoft.VisualStudio.Mac.LanguageServices.Razor/Microsoft.VisualStudio.Mac.LanguageServices.Razor.csproj]: This is a workaround for an MSBuild bug, see commit details.

File: korebuild-lock.txt (hunk 1 above)
Comment (rynowak) [reply korebuild-lock.txt]: Note: this is the release branch

Comment (rynowak) [reply korebuild-lock.txt]: I know @pranavkm signed off on this bad boy but can I get one of you to confirm that this is right for the release/2.1 branch? @natemcmaster @pranavkm  plz.

Comment (natemcmaster) [reply korebuild-lock.txt]: Normally the right way to update korebuild is `run.ps1 -update`. In this case, korebuild.json is misconfigured to point to the `dev` branch of build tools. Update it to say `"channel": "release/2.1"` and try `run.ps1 -update` again.

Comment (natemcmaster) [reply korebuild-lock.txt]: And no, build tools 2.1.0-preview2-15688 is not the right one to use. The latest release/2.1 build tools is 2.1.0-preview1-1005

See https://github.com/aspnet/BuildTools#latest-build

Comment (rynowak) [reply korebuild-lock.txt]: So does this need to be updated in all of our release branches then?

**Comment by rynowak** [comment]
Updated with the correct Korebuild

File: korebuild-lock.txt (hunk 1 above)
Comment (natemcmaster) [reply korebuild-lock.txt]: Yeah. I was planning to do it. Just waiting to get a build of the SDK that actually works. (blocked on https://github.com/aspnet/BuildTools/issues/562)

Comment (rynowak) [reply korebuild-lock.txt]: OK cool, I logged an issue in aspnet/Universe so it didn't get buried. Looks like you already found that though

**Comment by rynowak** [comment]
I'm planning to merge this tomorrow when I have access to corpnet to watch the CI.

**Comment by pranavkm** [comment]
@rynowak does this need to go in to preview1?

**Comment by pranavkm** [comment]
FYI - the dev branch picked up a new SDK which is causing tests to fail. I checked this change in to dev. See https://github.com/aspnet/Razor/commit/6d757988c9399af16f61dcf94c0f3f42df01642b

**Comment by rynowak** [comment]
> @rynowak does this need to go in to preview1?

image

Yes, as far as I know we are still waiting for preview1 to pick up a new SDK

**Comment by rynowak** [comment]
OK we finally have the new SDK. Updated this again, I plan to push after travis.

**Pull request merged**